cobigen-cli/cli/src/main/java/com/devonfw/cobigen/cli/CobiGenCLI.java

```diff
@@ -34,11 +35,18 @@ public static void main(String... args) {
     LOG.debug("Current working directory: {}", System.getProperty("user.dir"));
     CommandLine commandLine = new CommandLine(new CobiGenCommand());
     commandLine.registerConverter(Path.class, s -> Paths.get(s));
+    CommandLine.IExecutionExceptionHandler exceptionHandler = new CommandLine.IExecutionExceptionHandler() {
+      @Override
+      public int handleExecutionException(Exception e, CommandLine commandLine, CommandLine.ParseResult parseResult) throws Exception {
+        LOG.error("An error occurred while executing CobiGen: {}", e.getMessage());
+        if (LOG.isDebugEnabled()) e.printStackTrace();
+        return 1;
+      }
+    };
+    commandLine.setExecutionExceptionHandler(exceptionHandler);
     int exitCode = commandLine.execute(args);
-    if (exitCode == 0) {
-      LOG.info("Success");
-    } else {
-      LOG.info("Failed");
+    if (exitCode != 0) {
+      LOG.error("Cobigen terminated in an unexpected manner." + MessagesConstants.VERBOSE_HINT);
```

Comment:
Missing whitespace behind the dot

Reply:
Fixed in next commit.

Reply:
Shouldn't this be done like:
`LOG.error("Cobigen terminated in an unexpected manner. {}", MessagesConstants.VERBOSE_HINT);`